features/step_definitions/civil_journey_steps.rb

```diff
@@ -644,6 +659,7 @@
     :with_non_passported_state_machine,
     :provider_assessing_means,
     :with_policy_disregards,
+    # as we dont categorise on applicant side this traits should be removed, but I expect they will break many tests
```

Comment:
do we need to try this out to prevent faulty tests that could introduce false positives and negatives?

Reply:
I think these traits need to remain as we want to add the transactions to the application in this setup step so that the provider can categorise them